bin/windows-env.ps1

```diff
@@ -47,11 +53,9 @@ $curlPkg = "${curlVer}-${mingwVer}"
 $Wix35_VERSION = '3.5.2519.20130612'
 
 $env:PATH = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";" + [System.Environment]::GetEnvironmentVariable("Path","User")
-if ($arch -eq 32) {
-  $env:PATH = "C:\tools\mingw32\bin;" + $env:PATH
-}
-$env:PATH += [Environment]::GetFolderPath('ProgramFiles') + "\Git\cmd"
+@([Environment]::GetFolderPath('ProgramFiles') + "\Git\cmd";,
```

Comment:
Gives me: `Updated Path to C:\Program Files\Git\cmd; C:\cygwin64\home\Administrator\deps\ruby-2.1.7-x64-mingw32\bin; C:\cygwin64\home\Administrator\deps\openssl-1.0.2e-x64-windows\bin; C:\tools\mingw64\bin;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;c:\pstools;` after updating to put the semicolon in the quotes